### ISSUE TITLE
Please help to add About page into the nav bar

### ISSUE THREAD

**Issue by vietaws** [apvarun/showcase-hugo-theme#2]
Hi Varun,
Appreciate your effort to create this simple and fast hugo theme. I am writing this to ask ABOUT page into a nav bar, this is not an issue. Thank you for your hard work.

**Comment by apvarun** [comment]
@cloudmiracle Support for custom menu is now added

Please refer to [exampleSite/config.toml](https://github.com/apvarun/showcase-hugo-theme/blob/master/exampleSite/config.toml) for details on how to add a new menu

**Comment by vietaws** [comment]
thank you for your effort @apvarun. I can see the External Item on the menu. Can you help to keep the logo, nav bar and logo with SPA concept (dont open new link) in your free time. Thank you!

**Comment by BurgerhoutJ** [comment]
> thank you for your effort @apvarun. I can see the External Item on the menu. Can you help to keep the logo, nav bar and logo with SPA concept (dont open new link) in your free time. Thank you!

Is there any progress about this? I want to create a about/team page as well, but the nav bar has to be there.

**Comment by apvarun** [comment]
@cloudmiracle Could you rephrase your need here? I don't understand the SPA part in your query.

**Comment by apvarun** [comment]
Closing this due to inactivity
